docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -260,6 +263,7 @@ kind: OpenStackDataPlaneNodeSet
 metadata:
   name: openstack
 spec:
+  tlsEnabled: true
```

Comment:
As commented above this will default to true,  So we want an instruction to disable in case we are adopting from a non-tls environment.

Reply:
non tls-e that is.